openstack/objectstorage/v1/containers/results.go

```diff
@@ -96,11 +97,48 @@ func ExtractNames(page pagination.Page) ([]string, error) {
 	}
 }
 
+// GetHeader represents the headers returned in the response from a Get request.
+type GetHeader struct {
+	AcceptRanges     string    `mapstructure:"Accept-Ranges"`
+	BytesUsed        int64     `mapstructure:"X-Account-Bytes-Used"`
+	ContentLength    int64     `mapstructure:"Content-Length"`
+	ContentType      string    `mapstructure:"Content-Type"`
+	Date             time.Time `mapstructure:"-"`
```

Comment:
Because there's a lot of repeated headers, maybe we could embed a common Header struct?